flask_restplus/api.py

```diff
@@ -376,6 +381,14 @@ def documentation(self, func):
         self._doc_view = func
         return func
 
+    def specs(self, func):
+        '''A decorator to specify a view function for the specs'''
+        self._hide_specs_url = True
```

Comment:
Maybe better to default `self._spec_view` to `partial(apidocs.specs_for, self)`. Remove the `_hide_specs_url` so it is always available.

And always inline the swagger data.

I think this results in fewer options, and is closer to original functionality.